app/src/main/java/edu/rit/csh/agargiulo/Gatekeeper/HttpsConnector.java

```diff
@@ -59,6 +60,7 @@ private URI createUrl (String path)
 		try
 		{
 			postUri = URIUtils.createURI("https", POST_HOST, 0, path, null, null);
+            Log.wtf("CSH-Gatekeeper", POST_HOST);
```

Comment:
Is this strictly necessary? It's not indented correctly on top of the fact that it's always logging wtf level messages.